### PR TITLE
Updating dashboard job

### PR DESCRIPTION
**Pull request by TonioGela**

### CODE REVIEW

File: kubernetes/configs/prod/versions.sh [hunk 1]
```diff
@@ -25,4 +25,4 @@ JOB_DETAILS_PERSISTER_IMAGE_VERSION="1.0.11"
 JOB_TENANTS_CERTIFIED_ATTRIBUTES_UPDATER_IMAGE_VERSION="1.0.11"
 JOB_METRICS_REPORT_GENERATOR_IMAGE_VERSION="1.0.11"
 JOB_PADIGITALE_REPORT_GENERATOR_IMAGE_VERSION="1.0.11"
```
Comment (beetlecrunch) [inline kubernetes/configs/prod/versions.sh]: Why not moving also these versions to 1.0.12?

Comment (TonioGela) [reply kubernetes/configs/prod/versions.sh]: makes sense

File: kubernetes/configs/test/versions.sh [hunk 2]
```diff
@@ -26,4 +26,4 @@ JOB_DETAILS_PERSISTER_IMAGE_VERSION="1.0.11"
 JOB_TENANTS_CERTIFIED_ATTRIBUTES_UPDATER_IMAGE_VERSION="1.0.11"
 JOB_METRICS_REPORT_GENERATOR_IMAGE_VERSION="1.0.11"
 JOB_PADIGITALE_REPORT_GENERATOR_IMAGE_VERSION="1.0.11"
```
Comment (beetlecrunch) [inline kubernetes/configs/test/versions.sh]: same

Comment (TonioGela) [reply kubernetes/configs/test/versions.sh]: fixed

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**